docs/source/workflows/design_spaces.rst

```diff
@@ -284,3 +284,30 @@ We will require that formulations contain 2 ingredients, that no more than 1 sol
   )
 
   registered_design_space = project.design_spaces.register(design_space)
+
+Sampling from Design Spaces
+---------------------------
+
+To sample candidates from a registered design space, you can use the Citrine Python client as shown in the example below:
+
+.. code:: python
+
+   from citrine import Citrine
+   from citrine.jobs.waiting import wait_while_executing
+   from citrine.informatics.design_spaces.sample_design_space import SampleDesignSpaceInput
+
+   # Replace with your API key, host, project ID, and design space ID
+   session = Citrine(api_key="API_KEY", scheme="https", host="HOST", port="443")
+   project = session.projects.get("PROJECT_ID")
+   design_space = project.design_spaces.get("DESIGN_SPACE_ID")
```

Comment:
In the other examples on this page, we omit these lines to avoid confusion, and just refer to them by reasonable variable names. Please do the same here.